gcs-action/action.yml

```diff
@@ -0,0 +1,26 @@
+name: 'GCS Action'
+description: 'Enable easy upload of files and directories to GCS'
+author: 'GoogleCloudPlatform'
+inputs:
+  service_account_key:
+    description: 'GCP service account key'
+    required: true
+  folder-to-upload:
+    description: 'Folder to be uploaded'
+    required: false
+  file-to-upload:
+    description: 'File to be uploaded'
+    required: false
+  bucket-name:
+    description: 'GCS Bucket Name'
+    required: true
+  object-prefix:
+    description: 'GCS object prefix'
+    required: true
+  clear-existing-files-first:
```

Comment:
This feels like scope creep. If we want to have an action that deletes objects from a Cloud Storage bucket, we should make an action that does that (and only that). This feels unnecessarily magical and dangerous.

Reply:
A hearty +1 to the above. Let's keep it simple.

Reply:
Done, removed all deletion features